notebooks/mks_localiztion_test.ipynb

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Move the notebook to a test in Python. Write a short docstring for the test function to explain what the test does.